docs/platforms/python/configuration/releases.mdx

```diff
@@ -1,77 +1,53 @@
 - exceed 200 characters
 
-The value can be arbitrary, but we recommend either of these naming strategies:
-
-- **Semantic Versioning**: `package@version` or `package@version+build` (for example, `my.project.name@2.3.12+1234`)
-  - `package` is the unique identifier of the project/app (`CFBundleIdentifier` on iOS, `packageName` on Android)
-  - `version` is the semver-like structure `<major>.<minor?>.<patch?>.<revision?>-<prerelease?>` (`CFBundleShortVersionString` on iOS, `versionName` on Android)
-  - `build` is the number that identifies an iteration of your app (`CFBundleVersion` on iOS, `versionCode` on Android)
-- **Commit SHA**: If you use a version control system like Git, we recommend using the identifying hash (for example, the commit SHA, `da39a3ee5e6b4b0d3255bfef95601890afd80709`). You can let Sentry CLI automatically determine this hash for supported version control systems. Learn more in our [Sentry CLI](/product/cli/releases/#creating-releases) documentation.
-
-<Note>
+The value can be arbitrary, but we recommend [Semantic Versioning](https://semver.org/), [Calendar Versioning](https://calver.org/), or the Git commit SHA.
 
-Releases are global per organization; prefix them with something project-specific for easy differentiation.
+Releases are global per organization; prefix them with something project-specific (ex: `"myapp@1.0.0"`) for easy differentiation.
 
-</Note>
 
 The behavior of a few features depends on whether a project is using semantic or time-based versioning.
 
 - Regression detection
-- `release:latest`
+- Filter data by `release:latest`
 
-We automatically detect whether a project is using semantic or time-based versioning based on:
-
-- If ≤ 2 releases total: we look at most recent release.
-- If 3-9 releases (inclusive): if any of the most recent 3 releases is semver, project is semver.
-- If 10 or more releases: if any of the most recent 3 releases is semver, and 3 out of the most recent 10 releases is semver, then the project is semver.
+We automatically detect whether a project is using semantic or time-based versioning.
 
 ## Setting a Release
 
 <PlatformContent includePath="set-release" notateUnsupported />
 
-How you make the release name (or version) available to your code is up to you. For example, you could use an environment variable that is set during the build process or during initial start-up.
+How you make the `release` available to your code is up to you. For example, you could use an environment variable that is set during the build process or during initial start-up.
+
+The `release` can also be set by an environment variable `SENTRY_RELEASE`.
+
+If no `release` is set the Sentry SDK will try to guess it. If a Git repository is present it will take the Git SHA from the latest commit. If this did not work the SDK will check some environment variables used by hosting providers like `HEROKU_SLUG_COMMIT`, `SOURCE_VERSION`, `CODEBUILD_RESOLVED_SOURCE_VERSION`, `CIRCLE_SHA1`, `GAE_DEPLOYMENT_ID`.
```

Comment:
```suggestion
If no `release` is set, the Sentry SDK will try to guess it. If a Git repository is present, it will take the Git SHA from the latest commit. If this did not work, the SDK will check some environment variables used by hosting providers like `HEROKU_SLUG_COMMIT`, `SOURCE_VERSION`, `CODEBUILD_RESOLVED_SOURCE_VERSION`, `CIRCLE_SHA1`, and `GAE_DEPLOYMENT_ID`.
```